### PR TITLE
operators / it-academy.by / JC1-117-22 / Aliaksei Kharko / Done tasks 01-06

### PR DESCRIPTION
**Pull request by KharkoAO**
## Pull request state

Actual status for PR:
- [x] implemented
- [x] reviewed
- [x] fixed
- [ ] approved

### CODE REVIEW

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task01.java [hunk 1]
```diff
@@ -29,8 +29,7 @@ public static void main(String[] args) {
      * @param carryingCapacity грузоподъемность грузовиков
      */
     static int getNumberFullTrucks(int cargo, int carryingCapacity) {
-        //TODO
-        // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0;
+
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/operators/Task01.java:32]: "Violation: Useless blank lines
Solution: Remove blank lines"

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task02.java [hunk 2]
```diff
@@ -30,8 +30,7 @@ public static void main(String[] args) {
      * @param carryingCapacity грузоподъемность грузовиков
      */
     static int getAmountRemainingCargo(int cargo, int carryingCapacity) {
-        //TODO
-        // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0;
+
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/operators/Task02.java:33]: "Violation: Useless blank lines
Solution: Remove blank lines"

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task03.java [hunk 3]
```diff
@@ -35,8 +34,7 @@ public static void main(String[] args) {
      * @param depositAnnualPercentage ежегодный простой процент
      */
     static float getTotalDepositAmount(int depositAmount, int depositYears, int depositAnnualPercentage) {
-        //TODO
-        // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0.0f;
+
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/operators/Task03.java:37]: "Violation: Useless blank lines
Solution: Remove blank lines"

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task04.java [hunk 4]
```diff
@@ -33,8 +33,7 @@ public static void main(String[] args) {
      * @param milliseconds время (в миллисекундах) между срабатыванием датчика света и звука
      */
     static float getDistance(float milliseconds) {
-        //TODO
-        // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0.0f;
+
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/operators/Task04.java:36]: "Violation: Useless blank lines
Solution: Remove blank lines"

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task05.java [hunk 5]
```diff
@@ -34,8 +34,7 @@ public static void main(String[] args) {
      * @param weight - вес
      */
     static int calculateDeviationStandardWeight(int height, int weight) {
-        //TODO
-        // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0;
+
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/operators/Task05.java:37]: "Violation: Useless blank lines
Solution: Remove blank lines"

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task06.java [hunk 6]
```diff
@@ -28,8 +28,11 @@ public static void main(String[] args) {
      * @return время в формате 'D H:m:s', где D - дни, H - часы, m - минуты, s - секунды
      */
     static String getPlayingTime(int playingTimeInSeconds) {
-        //TODO
-        // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return null;
+        int s = playingTimeInSeconds % 60;
+        int m = (playingTimeInSeconds / 60) % 60;
+        int H = (playingTimeInSeconds / 3600) % 24;
+        int D = playingTimeInSeconds / 86400;
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/operators/Task06.java:34]: "Code smell: Magic Number
Solution: Named Constants"

Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/operators/Task06.java:34]: "Violation: 'nameVariable' was not used for variable naming
Solution: Rename variable"

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task05.java [hunk 7]
```diff
@@ -34,8 +34,7 @@ public static void main(String[] args) {
...
+        return weight - (height - 110);
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/operators/Task05.java:38]: "Code smell: Magic Number
Solution: Named Constants"

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task04.java [hunk 8]
```diff
@@ -33,8 +33,7 @@ public static void main(String[] args) {
...
+        return (SOUND_SPEED * (milliseconds / 3600000)) * 1000;
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/operators/Task04.java:37]: "Code smell: Magic Number
Solution: Named Constants"

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task03.java [hunk 9]
```diff
@@ -35,8 +34,7 @@ public static void main(String[] args) {
...
+        return depositAmount + ((depositAmount * (float)depositAnnualPercentage / 100) * depositYears);
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/operators/Task03.java:38]: "Code smell: Magic Number
Solution: Named Constants"

**Review by rakovets — CHANGES_REQUESTED** [review]
Need to fix

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task01.java (hunk 1 above)
Comment (KharkoAO) [reply src/main/java/com/rakovets/course/java/core/practice/operators/Task01.java]: this fixed

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task02.java (hunk 2 above)
Comment (KharkoAO) [reply src/main/java/com/rakovets/course/java/core/practice/operators/Task02.java]: this fixed

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task03.java (hunk 3 above)
Comment (KharkoAO) [reply src/main/java/com/rakovets/course/java/core/practice/operators/Task03.java]: this fixed

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task04.java (hunk 4 above)
Comment (KharkoAO) [reply src/main/java/com/rakovets/course/java/core/practice/operators/Task04.java]: this fixed

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task05.java (hunk 5 above)
Comment (KharkoAO) [reply src/main/java/com/rakovets/course/java/core/practice/operators/Task05.java]: this fixed

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task06.java (hunk 6 above)
Comment (KharkoAO) [reply src/main/java/com/rakovets/course/java/core/practice/operators/Task06.java]: this fixed

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task05.java (hunk 7 above)
Comment (KharkoAO) [reply src/main/java/com/rakovets/course/java/core/practice/operators/Task05.java]: this fixed

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task04.java (hunk 8 above)
Comment (KharkoAO) [reply src/main/java/com/rakovets/course/java/core/practice/operators/Task04.java]: this fixed

File: src/main/java/com/rakovets/course/java/core/practice/operators/Task03.java (hunk 9 above)
Comment (KharkoAO) [reply src/main/java/com/rakovets/course/java/core/practice/operators/Task03.java]: this fixed

**Review by rakovets — CHANGES_REQUESTED** [review]
Need to fix.
You need to add task from theme operators (ONLY)

**Pull request closed without merging**